dpypelines/s3_tar_received.py

```diff
@@ -46,56 +48,29 @@ def start(s3_object_name: str):
             )
         ) from err
 
-    # Check for the existence of a configuration file
+    # get_dataset_id() currently empty - returns "not-specified"
+    # To be updted once we know where the dataset_id can be extracted from
```

Comment:
can you add. that working from the s3 object name is an assumption, might be we do, might not be, tbd.